app/src/ui/app.tsx

```diff
@@ -1651,26 +1651,27 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
 
     if (selectedState.type === SelectionType.Repository) {
-      const externalEditorLabel = this.state.selectedExternalEditor
+      const externalEditorLabel = state.selectedExternalEditor
```

Comment:
These changes are really just a bit of cleanup. We had a variable holding state, so I just removed `this` from each usage of `state`.